### DISCUSSION TITLE
GoDNS doesn't do anything

### DISCUSSION THREAD

**Discussion by Tillerino** [TimothyYe/godns#235] [Q&A] [answered]
I adapted the example for DuckDNS like this:

```json
{
  "provider": "DuckDNS",
  "password": "",
  "login_token": "XXXXXXXXXXXXX",
  "domains": [
    {
      "domain_name": "www.duckdns.org",
      "sub_domains": ["XXXXXXXXXXX"]
    }
  ],
  "resolver": "8.8.8.8",
  "ip_urls": ["https://api64.ipify.org"],
  "ip_type": "IPv6",
  "interval": 60,
  "socks5_proxy": ""
}
```

When I start GoDNS with `godns -c duckdns.json`, it outputs the following and then just doesn't do anything:

```
INFO[0000] Creating DNS handler with provider: DuckDNS
INFO[0000] GoDNS started, starting the DNS manager...
```

The DNS record is not being updated and there are no error messages. Is there a verbose mode or something?

**Comment by TimothyYe** [comment]
Set `"debug_info": true`, it enables the verbose mode for debugging.

**Comment by TimothyYe** [comment, marked as answer]
BTW, it seems that you are using the IPv6 mode, I think you should set the `ipv6_urls` instead, not the `ip_urls`

**Comment by Tillerino** [comment]
Yes and yes! Thx for the quick help!
